tests/unit/providers/vector_io/remote/test_pgvector.py

```diff
@@ -0,0 +1,578 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import asyncio
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import numpy as np
+import pytest
+
+from llama_stack.apis.vector_dbs import VectorDB
+from llama_stack.apis.vector_io import Chunk, ChunkMetadata, QueryChunksResponse
+from llama_stack.providers.remote.vector_io.pgvector.config import PGVectorVectorIOConfig
+from llama_stack.providers.remote.vector_io.pgvector.pgvector import PGVectorIndex, PGVectorVectorIOAdapter
+
+PGVECTOR_PROVIDER = "pgvector"
```

Comment:
can we enable pgvector in here too - https://github.com/meta-llama/llama-stack/blob/1721aafc1fc13297d7db6c3bcb0c65344c6a7cd0/tests/unit/providers/vector_io/conftest.py#L32